README.md

```diff
@@ -25,6 +25,13 @@ The time is measured in milliseconds. The default is 1200000 (20 minutes).
 </ServiceWorkerUpdateNotify>
 ```
 
+The poll interval can also be configured in your `.ember-cli-buil.js` with:
```

Comment:
It's configured through `config/environment.js`, isn't it?

Reply:
Whoops. Yup